src/commands/Anime.js

```diff
@@ -0,0 +1,84 @@
+const { Command } = require('@dyno.gg/dyno-core');
+const superagent = require('superagent');
+
+
+class Anime extends Command {
+	constructor(...args) {
+		super(...args);
+
+		this.name = 'anime';
+		this.aliases = ['animu'];
+		this.module = 'Fun';
+		this.description = 'Get info about an anime';
+		this.usage = 'anime <anime>';
+		this.example = 'anime acchi kocchi';
+		this.cooldown = 5000;
+		this.expectedArgs = 1;
+	}
+
+	async execute({ message, args }) {
+		let msg = await this.sendMessage(message.channel, {
+			embed: {
+				color: 0x337fd5,
+				thumbnail: {
+					url: 'https://cdn.discordapp.com/emojis/397911964988342282.gif',
```

Comment:
Discord CDNs can't be trusted, as the content may dissapear without further notice, or someone may delete the emoji. 
I have uploaded that to our own CDN, mind chaning the link? https://cdn.dyno.gg/fun/discord_loading.gif